packages/components/src/combobox-control/index.tsx

```diff
@@ -36,16 +36,18 @@ import { useDeprecated36pxDefaultSizeProp } from '../utils/use-deprecated-props'
 
 const noop = () => {};
 
+interface DetectOutsideComponentProps {
+	onFocusOutside: ( event: FocusEvent ) => void;
```

Comment:
@ciampo `React.FocusEvent` or base `FocusEvent`?

Reply:
That should be a `React.FocusEvent`, judging by the code. It can also be verified by console.logging the value of `event`, which confirms that it in indeed a React synthetic event

Reply:
Done.